packages/client/src/builders/search/contentHighlightingBuilder.ts

```diff
@@ -0,0 +1,50 @@
+import { ContentHighlightProps, ContentSearchSettingsHighlightSettings, HighlightSettings2ContentContentHighlightPropsHighlightSettings2Limits, HighlightSettings2ContentContentHighlightPropsHighlightSettings2ResponseShape } from '../../models/data-contracts';
+
+export class ContentHighlightingBuilder {
+    private enabled: boolean = true;
+    private highlightable: ContentHighlightProps = {
+        $type: 'Relewise.Client.Requests.Shared.Highlighting.ContentHighlightProps, Relewise.Client',
+        displayName: false
+    }; 
+    private limit: HighlightSettings2ContentContentHighlightPropsHighlightSettings2Limits = {};
+    private shape: HighlightSettings2ContentContentHighlightPropsHighlightSettings2ResponseShape = {
+        includeOffsets: false
+    };
+
+    public enable(enabled: boolean): this {
```

Comment:
Would like the public method to be called `enabled(true)` or `enabled(false)` as this accepts a bools, it reads better

Reply:
Check!